File: lambda-test-csv/handler.js
"use strict";

const fs = require("fs");
const csv = require("csv-parser");

function getPeople() {
    return new Promise((resolve, reject) => {
        const people = [];

        try {
            fs.createReadStream("people.csv")
                .pipe(csv({ separator: "," }))
                .on("data", (line) => {
                    people.push(line);
                })
                .on("end", () => {
                    resolve(people);
                });
        } catch (ex) {
            reject(ex);
        }
    });
}

async function hello(event) {
    const people = await getPeople();

    console.log(people);

    return {
        statusCode: 200,
        body: JSON.stringify(
            {
                message: "Go Serverless v2.0! Your function executed successfully!",
                input: event,
            },
            null,
            2
        ),
    };
}

module.exports = {
    hello,
};

File: lambda-hello-env-vars/handler.py
import json
import os

def run(event, context):
    MY_VAR = os.environ['MY_VAR']

    body = {
        "message": "Go Serverless v1.0! Your function executed successfully!",
        "MY_VAR": MY_VAR
    }

    response = {
        "statusCode": 200,
        "body": json.dumps(body)
    }

    return response